wire-ios-data-model/Source/Model/UserClient/UserClient.swift

```diff
@@ -585,12 +585,8 @@ extension UserClient {
 
         proteusProvider.perform(
             withProteusService: { proteusService in
-                guard let sessionID = self.proteusSessionID else {
-                    return
-                }
-
                 do {
-                    let fingerprint = try proteusService.localFingerprint(forSession: sessionID)
+                    let fingerprint = try proteusService.fingerprint()
```

Comment:
Let's delete the `fingerprint` method from the proteus service.

Reply:
I've deleted `localFingerprint(forSession id: ProteusSessionID)` from the `ProteusServiceInterface` and renamed `fingerprint()`. Now we have 3 methods:
- `localFingerprint()` -> for self client
-` remoteFingerprint(forSession id: ProteusSessionID)` -> for other users and other clients of the self user
- `fingerprint(fromPrekey prekey: String)`